### PR TITLE
Internal improvement: Avoid combining distinct compilations in debugger tests

### PR DESCRIPTION
**Pull request by haltman-at**
Distinct compilations were being combined.  I got rid of this, and also simplified the setup.  Debugger tests will now run directly off of compiler output.

### CODE REVIEW

File: packages/debugger/test/helpers.js [hunk 1]
```diff
@@ -27,14 +28,14 @@ export async function prepareContracts(provider, sources = {}, migrations) {
...
-        optimizer: {enabled: false, runs: 200},
+        optimizer: { enabled: false, runs: 200 },
         evmVersion: "constantinople"
       }
     }
   };
 
   await addContracts(config, sources);
-  let {contractNames, files} = await compile(config);
+  let { contractNames, compilations: rawCompilations } = await compile(config);
```
Comment (eggplantzzz) [inline packages/debugger/test/helpers.js:38]: Oh interesting, I didn't know you could reference `compilations` this way.

File: packages/debugger/test/helpers.js [hunk 2]
```diff
@@ -134,38 +128,18 @@ export async function defaultMigrations(contractNames) {
...
-  const contractNames = collectedCompilationOutput.contractNames.filter(
-    (item, index) => {
-      return collectedCompilationOutput.contractNames.indexOf(item) === index;
-    }
-  );
-  return {
-    contractNames,
-    files: sourceIndexes
-  };
+  return { compilations, contractNames };
```
Comment (eggplantzzz) [inline packages/debugger/test/helpers.js:142]: Do you not need to adjust whatever code was previously calling this method? Or maybe it currently only uses `contractNames`?

Comment (eggplantzzz) [reply packages/debugger/test/helpers.js:142]: Oh nevermind, I see that you already did adjust it in this file!

**Review by eggplantzzz — APPROVED** [review]
Seems good to me.

File: packages/debugger/test/helpers.js (hunk 1 above)
Comment (haltman-at) [reply packages/debugger/test/helpers.js:38]: See your other comment. :)

File: packages/debugger/test/helpers.js (hunk 2 above)
Comment (haltman-at) [reply packages/debugger/test/helpers.js:142]: Yup, your two comments answer one another. :)

**Pull request merged**